packages/lit-task-scheduler/src/initializeTasks.ts

```diff
@@ -0,0 +1,14 @@
+import { Agenda } from '@hokify/agenda';
+import { handleTask as handleAutoTopUp, taskName as autoTopUpName } from 'lit-task-auto-top-up';
+
+const AUTO_TOP_UP_TASK_NAME = `daily ${autoTopUpName}`;
+export default async function initializeTasks(taskClient: Agenda) {
+  // Idempotent calls here only. Yay. :)
+  await taskClient.every('1 day', AUTO_TOP_UP_TASK_NAME);
```

Comment:
Running the task like this won't tie the execution time with the time we start this app? Do we want that?

Reply:
Good catch - I was meaning to set this but forgot.  Fixed; now it will run at 10PM GMT, a couple of hours before any existing NFT will expire since they always expire at midnight UTC.  WDYT?